api/e2etests/e2e_seed_data.py

```diff
@@ -0,0 +1,51 @@
+from app.settings.common import (
+    E2E_CHANGE_EMAIL_USER,
+    E2E_SIGNUP_USER,
+    E2E_USER,
+)

```

Comment:
erk, NO!! settings must be imported like: 

```
from django.conf import settings
```

then used as `settings.E2E_USER` for example.

Reply:
Right, done.